README.md

```diff
@@ -20,15 +20,15 @@ The projects `CREDENTIAL_CONFIG.yml` file contains a list of variables which nee
 git clone https://github.com/<forked_repository_org>/tower_dummy_credentials
 ```
 
-2. Change the variable values in the `CREDENTIAL_CONFIG.yml` file. A list of all variables that need to be changed along with their usage can be found [HERE](VARIABLES.md).
+3. Make a copy of the `CREDENTIAL_CONFIG_TEMPLATE.yml` file and set the variables for your tower instance in the copied file `cp CREDENTIAL_CONFIG_TEMPLATE.yml CREDENTIAL_CONFIG.yml` . A list of all variables that need to be changed along with their usage can be found [HERE](VARIABLES.md).
 
-3. From the projects root directory, run the `bootstrap.yml` playbook, specifying the path to the `CREDENTIAL_CONFIG.yml` file.
+4. From the projects root directory, run the `bootstrap.yml` playbook, specifying the path to the `CREDENTIAL_CONFIG.yml` file.
 
 ```bash
-ansible-playbook -i ./inventories/hosts bootstrap.yml --extra-vars='@CREDENTIAL_CONFIG.yml'
+ansible-playbook -i ./inventories/hosts bootstrap.yml --extra-vars='@CREDENTIAL_CONFIG.yml
 ```
 
-4. Sensitive information is encrypted now, push to the GitHub (Despite the encryption, we still recommend pushing to the private repository).
+5. Sensitive information is encrypted now, push to the GitHub (Despite the encryption, we still recommend pushing to the private repository).
```

Comment:
Will need to add an additional step to say to run a `git add CREDENTIAL_CONFIG.yml` to include the newly generated file